### ISSUE TITLE
Imagenet VIT GLU ooms

### ISSUE THREAD

**Issue by priyakasimbeg** [mlcommons/algorithmic-efficiency#654]
Imagenet ViT GLU OOMs.

## Description

Logs: [link](https://gist.github.com/priyakasimbeg/caecc75edc95c64a415ee34cc7caefdd)

## Steps to Reproduce
```
python3 submission_runner.py --framework=jax --workload=imagenet_vit_glu --submission_path=reference_algorithms/target_setting_algorithms/jax_nadamw.py --tuning_search_space=reference_algorithms/target_setting_algorithms/imagenet_vit_glu/tuning_search_space.json --data_dir=/data/imagenet/jax --num_tuning_trials=1 --experiment_dir=/experiment_runs --experiment_name=variants_target_setting/study_0 --overwrite=true --save_checkpoints=false --num_tuning_trials=1 --rng_seed=3377931912 --max_global_steps=186666 --imagenet_v2_data_dir=/data/imagenet/jax 2>&1 | tee -a /logs/imagenet_vit_glu_jax_02-15-2024-09-19-44.log
```

**Comment by priyakasimbeg** [comment]
@pomonam thanks for working on this! Just checking if there are any updates here.

**Comment by pomonam** [comment]
I submitted a fix at #676 and confirmed that reducing the batch size works.
